index.html.in

```diff
@@ -183,11 +185,11 @@
                           <div class="form-group">
                               <div id="translateText">
                                   <div class="col-sm-6">
-                                      <textarea class="form-control" id="originalText" rows="15" spellcheck="false"></textarea>
-                                      <button class="btn clearButton"><i class="fa fa-times"></i></button>
+                                      <textarea class="form-control" id="originalText" rows="15" spellcheck="false" aria-label="originalText"></textarea>
+                                      <button class="btn clearButton" aria-label="clear"><i class="fa fa-times"></i></button>
                                   </div>
                                   <div class="col-sm-6">
-                                      <textarea class="form-control" id="translatedText" rows="15" spellcheck="false" readonly></textarea>
+                                      <textarea class="form-control" id="translatedText" rows="15" spellcheck="false" aria-label="translatedText" readonly></textarea>
```

Comment:
If I understand correctly, `aria-label` should be proper human readable words. e.g. "Translated Text"

Reply:
(this comment applies to the other attributes above)